.circleci/config.yml

```diff
@@ -327,6 +297,28 @@ jobs:
       - store_artifacts:
           path: /go/out/istio-sidecar.deb
 
+  ## CONTAINER BASED LINT. mukai's linter container does not have all
```

Comment:
is there an issue for this? we should be using a pinned gometalinter

Reply:
this is already using a pinned container.. @jmuk has it stored in his docker image

Reply:
Not sure about the intent of this comment, but if something is missing in the docker image, please update it. Apparently it doesn't have to be my one at all.

For the background, it used to be a docker image under istio-testing (built by @kyessenov ), I just replaced this by my own image when I updated some linters, since I don't have write access to istio-testing (see https://github.com/istio/istio/pull/1798).